client/api_challenge.go

```diff
@@ -12,6 +16,8 @@ import (
 
 type Challenge interface {
 	GetChallengeInfo(ctx context.Context, info types.ChallengeInfo) (types.ChallengeResult, error)
+	SubmitChallenge(ctx context.Context, challengerAddress, spOperatorAddress, bucketName, objectName string, randomIndex bool, segmentIndex uint32, txOption *gnfdsdktypes.TxOption) (*sdk.TxResponse, error)
```

Comment:
Can users submit challenges now? Please double confirm cc @forcodedancing

Reply:
If we don't supports currently, delete these interfaces.

Reply:
because relayer and challenger are also going to use this sdk.

Reply:
yes, will add